Makefile

```
@@ -1,162 +1,158 @@
-
-test-ci:
-	NODE_ENV=production $(MAKE) compile
-	@echo "$(GREEN)================= Running production tests =================$(RESET)"
-	SINGLE_RUN=true node ./scripts/runTestServer.js
-	node_modules/.bin/nyc report --temp-directory ./test/tmp --reporter=html --reporter=lcov --reporter=text
+			make test
 
-generate-images-local: compile-dev
-	@echo "$(GREEN)================== Generating test images ==================$(RESET)"
-	node scripts/runTestServer.js
+test-ci: # Run the tests (for use in the continuous integration environment)
+	@SINGLE_RUN=true NODE_ENV=production $(MAKE) test
+	@node_modules/.bin/nyc report --temp-directory ./test/tmp --reporter=html --reporter=lcov --reporter=text
 
-generate-images:
-	docker run \
+generate-images: ## Generate the screenshots to run the visual diff validation tests
+	@docker run \
 		-it \
 		--rm \
 		--name node \
 		-v "$(PWD):/usr/src/app" \
 		-w /usr/src/app \
 		-e SINGLE_RUN=true \
 		-e GENERATE_NEW_IMAGES=true \
-		-p 8080:8080 \
 		circleci/node:10-browsers \
-			make generate-images-local
-	$(MAKE) generate-mocks
-
-generate-mocks:
-	$(NODE) ./scripts/createMockJson.js
-
-lint:
-	node_modules/.bin/madge packages/entryPoints/index.ts --circular --warning
-	node_modules/.bin/madge packages --orphans --extensions ts --exclude '.+\.d.ts|.+/dist\/.+'
-	node_modules/.bin/tslint --project tsconfig.json
+			make test
 
-lint-fix:
-	node_modules/.bin/tslint --project tsconfig.json --fix
-	node_modules/.bin/prettier --write 'packages/**/*.{ts,tsx}'
-	node_modules/.bin/prettier --write 'packages/decentraland-ecs/types/dcl/index.d.ts'
+generate-mocks: public/local-ipfs/mappings ## Build a fake "IPFS" index of all the test scene mappings
 
-watch: export NODE_ENV=development
-watch: compile-dev
-	@echo "$(GREEN)=================== Watching file changes ==================$(RESET)"
-	$(MAKE) only-watch
-
-
-FILE=-100.*
-watch-single:
-	@echo '[{"name": "Debug","kind": "Webpack","file": "public/test-parcels/$(FILE)/game.ts","target": "web"}]' > build.single-debug.json
-	@node_modules/.bin/concurrently \
-		-n "entryPoints,debug-scene,server" \
-			"$(PARALLEL_COMPILER) build.entryPoints.json --watch" \
-			"$(PARALLEL_COMPILER) build.single-debug.json --watch" \
-			"node ./scripts/runTestServer.js --keep-open"
+PARCEL_SCENE_JSONS := $(wildcard public/test-scenes/*/scene.json)
+ECS_SCENE_JSONS := $(wildcard public/ecs-scenes/*/scene.json)
+HELLMAP_SCENE_JSONS := $(wildcard public/hell-map/*/scene.json)
+SCENE_JSONS := $(PARCEL_SCENE_JSONS) $(ECS_SCENE_JSONS) $(HELLMAP_SCENE_JSONS)
 
-watch-single-no-server:
-	@echo '[{"name": "Debug","kind": "Webpack","file": "public/test-parcels/$(FILE)/game.ts","target": "web"}]' > build.single-debug.json
-	@node_modules/.bin/concurrently \
-		-n "entryPoints,debug-scene" \
-			"$(PARALLEL_COMPILER) build.entryPoints.json --watch" \
-			"$(PARALLEL_COMPILER) build.single-debug.json --watch"
+public/local-ipfs/mappings: ./scripts/createMockJson.js $(SCENE_JSONS)
```

I'd argue to remove local-ipfs directory as the first step of this target, as we've had issues with preexistent contents for scenes that were later removed.
```suggestion
public/local-ipfs/mappings: ./scripts/createMockJson.js $(SCENE_JSONS)
	rm -r ./public/local-ipfs
```